pkg/pillar/zedbox/zedbox.go

```diff
@@ -153,6 +293,16 @@ func runService(serviceName string, sep entrypoint, inline bool) int {
 	if inline {
 		log.Functionf("Running inline command %s args: %+v",
 			serviceName, arguments)
+
+		// Init and process global config for the forced GC parameters
+		go initAndProcessSubGlobalConfig(serviceName)
+
+		// Every separate zedbox application (e.g. pillar, tpmmgr,
+		// upgradeconverter, etc - those which are inlined) listen to
+		// memory pressure events and try to be polite calling garbage
+		// collector
+		go readMemoryPressureEvents()
+
```

Comment:
This code is in the wrong branch - inline are for the commands which terminate.

And wouldn't it make more sense to put this code in one of the agents like watcher or nodeagent?
(watcher is already looking at memory usage and issues logs for significant changes.)

Reply:
Right, standalone commands. Are all of them terminate immediately? The idea is that if they stay alive for a while, then probably we can squeeze something from them calling GC. But now I feel that this is an overkill, and putting the handler into zedbox should be enough.